common/common.go

```diff
@@ -74,6 +74,7 @@ const (
 	IPv6Len    = 40
 	ICMPLen    = 8
 	TCPMinLen  = 20
+	TCPDataOff = 0x50
```

Comment:
It is actually TCPMinDataOffset

Reply:
I agree.

Reply:
If you start to add new constants not connected with length, can you
1) start const section
2) add constants
0x45 // Ipv4, IHL = 5 (min header len) - 610 line
SwapBytesUint32(0x60 << 24) // IP version - 632 line
and others.